### PR TITLE
Refactor update bounds method.

### PR DESCRIPTION
**Pull request by robertocp25**
Refactor update bounds method so we have one unified version with the information needed.

### CODE REVIEW

File: packages/atlas/lib/src/padding.dart [hunk 1]
```diff
@@ -0,0 +1,43 @@
+class Padding {
```
Comment (edsondiasalves) [inline packages/atlas/lib/src/padding.dart]: Instead of creating a new class for padding, you can use the existing EdgeInsets from the flutter framework

**Review by edsondiasalves — DISMISSED** [review]

File: packages/atlas/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # atlas
 
+## v0.14.0 (2021-01-14)
```
Comment (oguibueno) [inline packages/atlas/CHANGELOG.md:3]: missing pubspec version update

File: packages/atlas/lib/src/bounding_box_data.dart [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+/// Represents all information need to create the boundingBox
+class BoundingBoxData {
+  /// A latitude/longitude aligned rectangle `LatLngBounds`
+  final LatLngBounds bounds;
+
+  /// Optional 2D rectangle based on a Origin and Size `Rectangle2D`
+  final Rectangle2D rectangle2d;
+
+  /// Optional padding that should be apply to the bounding box
```
Comment (oguibueno) [inline packages/atlas/lib/src/bounding_box_data.dart]:
```suggestion
  /// Optional padding that should be applied to the bounding box
```

File: packages/atlas/lib/src/rectangle_2d.dart [hunk 4]
```diff
@@ -0,0 +1,35 @@
+///Represents a virtual rectangle of the screen
```
Comment (oguibueno) [inline packages/atlas/lib/src/rectangle_2d.dart]:
```suggestion
/// Represents a virtual rectangle of the screen
```

**Review by edsondiasalves — DISMISSED** [review]

File: packages/atlas/lib/src/rectangle_2d.dart [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+class Rectangle2D {
+  /// Represents the X-axis coordinate of the origin point
+  final double originX;
+
+  /// Represents the Y-axis coordinate of the origin point
```
Comment (oguibueno) [inline packages/atlas/lib/src/rectangle_2d.dart]:
```suggestion
  /// Represents the Y-axis point of the origin coordinate
```

File: packages/atlas/lib/src/rectangle_2d.dart [hunk 6]
```diff
@@ -0,0 +1,35 @@
+///Represents a virtual rectangle of the screen
+class Rectangle2D {
+  /// Represents the X-axis coordinate of the origin point
```
Comment (oguibueno) [inline packages/atlas/lib/src/rectangle_2d.dart]:
```suggestion
  /// Represents the X-axis point of the origin coordinate
```

**Review by oguibueno — APPROVED** [review]

**Review by edsondiasalves — APPROVED** [review]

**Pull request merged**